### PR TITLE
Ta python module

### PR DESCRIPTION
**Pull request by asadchev**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -406,3 +406,9 @@ ADD_CUSTOM_TARGET(release
 
 feature_summary(WHAT ALL
                 DESCRIPTION "=== TiledArray Package/Feature Info ===")
+
+option(TA_PYTHON "Build TA python module" ON)
```
Comment (evaleev) [inline CMakeLists.txt:410]: should build this by default only if `BUILD_SHARED_LIBS=ON`? see `redefaultable_option`

**Comment by evaleev** [comment]
need to add TA copyright/license headers to the files created by you

File: CMakeLists.txt (hunk 1 above)
Comment (asadchev) [reply CMakeLists.txt:410]: I m actually going to make it off by default as it could create some issues in mpqc build

**Comment by evaleev** [comment]
Ok

**Pull request merged**